config/pre-cache/job.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: pre-cache
+spec:
+  backoffLimit: 0
+  activeDeadlineSeconds: 14400
+  template:
+    metadata:
+      name: pre-cache
+    spec:
+      containers:
+      - name: pre-cache-container
+        image: quay.io/vgrinber/pre-cache:0.0.3
+        env:
+        - name: pull_timeout
+          value: "14400"
+        - name: config_volume_path
+          value: /etc/config
+        command:
+        - /bin/bash
```

Comment:
Can you make a single entry point executable (can be a shell script) which includes all the commands specified here. That allows future versions of the container to update this logic without requiring an update to the Jobs.

Reply:
Done